pkgs/development/tools/jira_cli/default.nix

```diff
@@ -28,5 +28,9 @@ in
       homepage = "https://github.com/alisaifee/jira-cli";
       maintainers = with maintainers; [ nyarly ];
       license = licenses.mit;
+      # Project is unmaintained
+      # https://github.com/alisaifee/jira-cli#this-project-needs-a-new-owner
+      # Requires suds-jurko which was removed (due to the usage of use_2to3)
+      broken = true;
```

Comment:
Would be nice to include a comment with current date.

Reply:
SUre, done.